kotoed-server/src/main/kotlin/org/jetbrains/research/kotoed/data/api/Data.kt

```diff
@@ -88,9 +87,31 @@ object Code {
                 val fromLine: Int? = null,
                 val toLine: Int? = null) : Jsonable
         data class ReadResponse(val contents: String, val status: CloneStatus) : Jsonable
-        data class ListRequest(val submissionId: Int) : Jsonable
-        data class DiffRequest(val submissionId: Int) : Jsonable
-        data class DiffResponse(val diff: List<DiffJsonable>, val status: CloneStatus) : Jsonable
+        data class ListRequest(val submissionId: Int, val diffBase: DiffRequest.DiffBase) : Jsonable
```

Comment:
Why do we need `diffBase` for list request?

Reply:
Probably forgot to remove it, will do.